JSDemos/Demos/Popup/Overview/Angular/description.md

```diff
@@ -1,10 +1,34 @@
 
-- **Show and hide the Popup**      
-Use two-way binding to define the [visible](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#visible) property and manipulate its value directly. If the Popup should be hidden on outside clicks, enable the [hideOnOutsideClick](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#hideOnOutsideClick) property.
+- Built-in close button    
+Enable the [showCloseButton](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#showCloseButton) property to display the close button in the Popup's [title area](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#showTitle).
+
+- Custom close button    
+This demo shows how to add custom buttons to the Popup. One of them uses an [onClick](/Documentation/ApiReference/UI_Components/dxButton/Configuration/#onClick) handler to set the `popupVisible` variable to `false`, and the Popup becomes invisible. To learn more, follow the **Configure the Popup** topic below.
+
+- On outside click    
+Users can also hide the Popup on outside clicks, if you enable the [hideOnOutsideClick](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#hideOnOutsideClick) property.
+
+## Configure the Popup
+
+The Popup inner area is divided into two parts: title area and content area.
+
+- Content area       
+To fill the Popup with content, put a markup inside the component. You can also specify a toolbar at the bottom of the content area. To do this, declare the [toolbarItems](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/toolbarItems/) in the markup as shown in this demo. Set each item's [toolbar](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/toolbarItems/#toolbar) property to `bottom`. To learn more about toolbar configuration, refer to the following tutorial: [Getting Started with Toolbar](/Documentation/Guide/UI_Components/Toolbar/Getting_Started_with_Toolbar/)
+
+- Title area       
+Set [showTitle](Documentation/ApiReference/UI_Components/dxPopup/Configuration/#showTitle) to `true` and use the [title](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#title) property to specify the caption. You can also add a toolbar in the title area. Set each [toolbarItems](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/toolbarItems/) item's [toolbar](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/toolbarItems/#toolbar) property to `top`. If you want to add an item to the overflow menu as shown in this demo, set its [locateInMenu](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/toolbarItems/#locateInMenu) to `always`.
```

Comment:
"toolbarItems item's" - that doesn't read well because of repeated word. 
"set its locateInMenu" - The pronoun (its) is too far from the noun it refers to. Maybe "its" refers to "demo", or "menu"... It's better to be more specific in this case. 

```suggestion
Set [showTitle](Documentation/ApiReference/UI_Components/dxPopup/Configuration/#showTitle) to `true` and use the [title](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#title) property to specify the caption. You can also add a toolbar to the title area. Add [toolbarItems](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/toolbarItems/) markup and set each item's [toolbar](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/toolbarItems/#toolbar) property to `top`. If you want to display an item in the overflow menu, as shown in this demo, set the item's [locateInMenu](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/toolbarItems/#locateInMenu) to `always`.
```